CHANGELOG.rst

```diff
@@ -1,5 +1,35 @@
 .. _changelog:
 
+0.68.1
+------
+
+This version of Renku introduces several bug fixes for the backend services. It also makes DOI
+data connectors available in the data services search. Note though that the frontend is not yet using
+this new version of the search. The switch will be done most likely in the next release.
+
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🐞 Bug Fixes**
+
+- **Data services**: In some cases the namespace and slug of similar but unrelated data connectors
```

Comment:
Note: the Data Connector was created and linked to the wrong project.

Reply:
Ok I will change it. It wasn't clear to me what the consequences are.